Integrations/integration-ServiceNow.yml

```diff
@@ -96,11 +96,22 @@ script:
         }
     };
 
-    var getBody = function() {
+    var getBody = function(templateName) {
         var body = {};
+        var tmpRes;
+
+        if (templateName){
```

Comment:
checking a !== undefined can be replaced with a, cause if a has a value if (a) return true, else false (also when a is false or empty string)